tests/src/integration/common/wskdeploy.go

```diff
@@ -47,17 +47,17 @@ func (wskdeploy *Wskdeploy) RunCommand(s ...string) ([]byte, error) {
 }
 
 func (wskdeploy *Wskdeploy) Deploy(manifestPath string, deploymentPath string) ([]byte, error) {
-	return wskdeploy.RunCommand("-m", manifestPath, "-d", deploymentPath)
+	return wskdeploy.RunCommand("-i ", "false", "-m", manifestPath, "-d", deploymentPath)
```

Comment:
@lzbj I am getting this error when I run wskdeploy with -i false, are you sure these tests are successful?

```
./wskdeploy -i false -m tests/src/integration/flagstests/manifest.yaml 
Error: unknown command "false" for "wskdeploy"
Run 'wskdeploy --help' for usage.
2017/08/21 16:49:19 unknown command "false" for "wskdeploy"
```